src/Http/Cookie/CookieCryptTrait.php

```diff
@@ -0,0 +1,184 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @since         3.1.6
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\Http\Cookie;
+
+use Cake\Utility\Security;
+use RuntimeException;
+
+/**
+ * Cookie Crypt Trait.
+ *
+ * Provides the encrypt/decrypt logic for the CookieComponent.
+ *
+ * @link http://book.cakephp.org/3.0/en/controllers/components/cookie.html
+ */
+trait CookieCryptTrait
+{
+
+    /**
+     * Valid cipher names for encrypted cookies.
+     *
+     * @var array
+     */
+    protected $_validCiphers = ['aes', 'rijndael'];
```

Comment:
This property came from a trait I've changed meanwhile. I could change it now because the trait has changed as well. So what do we do with methods coming from traits when we're changing the code? For example, you create a new class or refactor a class and remove all prefixes in this class, now you've got a trait that still uses them and you end up with mixed styles.

My last update was that we **don't** use the prefixes anymore. This would mean we would have to remove them *everywhere* in the core as well for 3.5 to get this consistent.

I would prefer to keep them but what really drives me crazy is to discuss this issue in every PR. 😞